res_partner_attributes_add_BN/i18n/res_partner_attributes_add_BN.pot

```diff
@@ -1,20 +1,20 @@
 # Translation of OpenERP Server.
```

Comment:
Careful, the .pot should not have translations. They are the files that are exported from Odoo as templates.